awscli/paramfile.py

```diff
@@ -37,6 +37,10 @@
     'cloudformation.update-stack.stack-policy-url',
     'cloudformation.set-stack-policy.stack-policy-url',
 
+    'custom.cp.website-redirect',
```

Comment:
Yeah, let's get the command lineage PR merged so we can update this PR to avoid `custom`.